### PR TITLE
Uppy Serverless

### PR DESCRIPTION
**Pull request by dcflow**

### CODE REVIEW

**Comment by kvz** [comment]
I'll let @ifedapoolarewaju have the final say but this looks really good! Awesome stuff @dcflow!

File: src/serverless/app.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+'use strict'
```
Comment (ifedapoolarewaju) [inline src/serverless/app.js]: can `src/serverless` be moved to `src/standalone/serverless`? Since it is also an example of standalone usage?

Comment (ifedapoolarewaju) [reply src/serverless/app.js]: also is it possible to not rewrite this entire file, but reuse `src/standalone/index.js` instead?

**Review by ifedapoolarewaju — COMMENTED** [review]
@dcflow thank you very much for the PR! I added a few notes, please let me know what you think.

File: src/serverless/app.js (hunk 1 above)
Comment (dcflow) [reply src/serverless/app.js]: @ifedapoolarewaju can you check is it ok?

Comment (ifedapoolarewaju) [reply src/serverless/app.js]: yes, it's ok for the first issue, how about the second one?

> also is it possible to not rewrite this entire file, but reuse src/standalone/index.js instead?

Comment (dcflow) [reply src/serverless/app.js]: Yeah I tried that a couple of times it works locally but not on Lambda .. maybe I'm missing some config.
You can save up some lines on the main file, but you have to change the way the folders are structured.
```
// handler.js

const awsServerlessExpress = require('aws-serverless-express')
const awsServerlessExpressMiddleware = require('aws-serverless-express/middleware')

const { app } = require('../index')

app.use(awsServerlessExpressMiddleware.eventContext())

const server = awsServerlessExpress.createServer(app)

exports.uppy = (event, context) =>awsServerlessExpress.proxy(server, event, context)
```
Anyway when I get to check more about `sls` and `lambda` configs I'll upload another version.

Comment (ifedapoolarewaju) [reply src/serverless/app.js]: @dcflow alright, I'll merge this for now, and create an issue to refactor it so we don't forget about it. Thank you for the PR 🙂

**Pull request merged**